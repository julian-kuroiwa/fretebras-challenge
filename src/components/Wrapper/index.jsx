import React from 'react';

import { Container } from './style';

const Wrapper = ({ children, className }) => (
  <Container className={className}>{children}</Container>
);

export default Wrapper;
