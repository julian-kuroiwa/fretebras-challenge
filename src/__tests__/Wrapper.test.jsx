import { shallow } from 'enzyme';
import React from 'react';

import Wrapper from '../components/Wrapper';

describe('Wrapper', () => {
  it('should match snapshot', () => {
    const component = shallow(<Wrapper />);

    expect(component).toMatchSnapshot();
  });

  it('should render a children passed', () => {
    const component = shallow(
      <Wrapper>
        <h1>Hello Test</h1>
      </Wrapper>,
    );

    expect(component.find('h1').text()).toBe('Hello Test');
  });
});
