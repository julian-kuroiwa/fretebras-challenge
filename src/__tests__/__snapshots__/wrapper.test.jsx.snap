// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Wrapper should match snapshot 1`] = `<styled.div />`;
